lib/pltr/v2/reducers/tours.js

```diff
@@ -9,57 +9,61 @@ export default function tours(state = INITIAL_STATE, action) {
       const newState = Object.assign({}, INITIAL_STATE)
       newState.feature = action.payload //an object with feature: 'string', featureId: int
       newState.showTour = true
-      return { ...state, ...newState }
+      return newState
     //main tour callback
     case TOUR_NEXT: {
       let transitionSteps = []
       let endStep
-      state.b2bTransition = false
-      //switch based on which tour is running -> to determine when the
-      //tour callback should delay until next step's component is mounted
-      switch (state.feature.name) {
+      const newState = Object.assign({}, state)
+
+      //ensure the tour runs at the next step
+      newState.loading = false
+      newState.run = true
+
+      //determines whether the tour callback moves on when 'next' is clicked or when a specific app component is clicked
+      switch (newState.feature.name) {
         case 'acts':
-          transitionSteps = [0, 4, 5, 6]
-          endStep = 9
+          transitionSteps = [0,4,5,6,7]
+          endStep = 8
           break
         default:
-          state.transitioning = false
-      }
-      //delay the next step until component associated with the step is mounted
-      if (transitionSteps.includes(state.stepIndex)) {
-        state.transitioning = true
-        //need to anticipate back-to-back delays
-        if (transitionSteps.includes(state.stepIndex + 1)) {
-          state.b2bTransition = true
-        }
-      } else {
-        state.transitioning = false
+          newState.transitioning = false
       }
+
       //increment or decrement stepIndex
       if (action.payload === 'prev') {
-        state.stepIndex = state.stepIndex - 1
+        newState.stepIndex = newState.stepIndex - 1
       } else {
-        state.stepIndex = state.stepIndex + 1
+        newState.stepIndex = newState.stepIndex + 1
       }
-      //ensure the tour runs at the next step
-      state.loading = false
-      state.run = true
+
+      newState.transitioning = false
+
+      //delay the next step until component associated with the step is mounted
+      if (transitionSteps.includes(newState.stepIndex)) {
+        newState.transitioning = true
+      } 
+
       //stop the tour at the appropriate endStep,
       //store that they have completed this tour,
       //clear the feature so the tour doesn't run again unless manually started
-      if (state.stepIndex === endStep) {
-        state.toursTaken[`${state.feature.name}`] = true
-        state.feature = {}
-        state.run = false
+      if (newState.stepIndex === endStep) {
+        newState.toursTaken.length ? newState.toursTaken = [...newState.toursTaken,newState.feature.name] : [newState.feature.name]
```

Comment:
It seems like we don't use the toursTaken property (I searched the repo with your branch checked out).
Is that intentional?